<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:app.ico</normaloff>:app.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxPack">
     <property name="title">
      <string>In the case of file conflict when packing a PBO:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbPackOption1">
        <property name="text">
         <string>Abort the operation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPackOption2">
        <property name="text">
         <string>Save the PBO as a copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPackOption3">
        <property name="text">
         <string>Owerwrite the existing file</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxUnpack">
     <property name="title">
      <string>In the case of file conflict when unpacking a PBO:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbUnpackOption1">
        <property name="text">
         <string>Skip the conflicting file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbUnpackOption2">
        <property name="text">
         <string>Save the extracted file as a copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbUnpackOption3">
        <property name="text">
         <string>Overwrite the existing file</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOpComplete">
     <property name="title">
      <string>When pack/unpack operation complete:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbOpCompleteOption2">
        <property name="text">
         <string>Keep the window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbOpCompleteOption1">
        <property name="text">
         <string>Close the window</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
